### PR TITLE
Build OpenSearch 2.0 daily and OpenSearch Dashboards 1.2.0 half-hourly.

### PR DESCRIPTION
**Pull request by dblock**
Signed-off-by: dblock <dblock@amazon.com>

### Description

I'd like to start building OpenSearch 2.0 daily, and hopefully close https://github.com/opensearch-project/opensearch-build/issues/74 soon. 
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: jenkins/opensearch/Jenkinsfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ pipeline {
     triggers {
       parameterizedCron '''
         H */2 * * * %INPUT_MANIFEST=1.2.0/opensearch-1.2.0.yml
+        0 1 * * * %INPUT_MANIFEST=2.0.0/opensearch-2.0.0.yml
```
Comment (peterzhuamazon) [inline jenkins/opensearch/Jenkinsfile:6]: It is not recommended to run workflows at the exact time in Jenkins.
Suggest H 1 * * * if you want to run the task between 1 and 2 every day for 2.0.0.

Comment (peterzhuamazon) [reply jenkins/opensearch/Jenkinsfile:6]: If you want to run 1:00 to 1:10 use H(0-10) * * * *.

Comment (dblock) [reply jenkins/opensearch/Jenkinsfile]: Updated.

**Comment by dblock** [comment]
Updated this to also build OpenSearch Dashboards 1.2.0, cc: @kavilla.

**Review by peterzhuamazon — APPROVED** [review]

**Review by peternied — APPROVED** [review]

**Pull request merged**